src/core/engines/Cesium/Feature/Raster/index.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import { Meta, Story } from "@storybook/react";
+
+import { engine } from "../..";
+import Component, { Props } from "../../../../Map";
+
+export default {
+  title: "core/engines/Cesium/Feature/Raster",
+  component: Component,
+  parameters: { actions: { argTypesRegex: "^on.*" } },
+} as Meta;
+
+const Template: Story<Props> = args => <Component {...args} />;
+
+export const Default = Template.bind([]);
+Default.args = {
+  engine: "cesium",
+  engines: {
+    cesium: engine,
+  },
+  ready: true,
+  layers: [
+    {
+      id: "l",
+      type: "simple",
+      data: {
+        type: "wms",
+        url: "",
+      },
+      raster: {
+        layers: [""],
```

Comment:
empty url?

```suggestion
      data: {
        type: "wms",
        url: "https://gibs.earthdata.nasa.gov/wms/epsg3857/best/wms.cgi",
        layer: "IMERG_Precipitation_Rate"
      },
      raster: {},
```

Note: that data is open data so you can use it in storybook

Reply:
It is fixed
[test: fix storybook data](https://github.com/reearth/reearth-web/pull/387/commits/141346a1bd72b9e52dcb0ccd0f5b4d69b20fcef1)